website/tests/test_main.py

```
@@ -131,86 +131,139 @@ def test_signup(self):
+
+    # 2️⃣ Start Selenium & go to login page
+    login_url = f"{self.live_server_url}/accounts/login/"
+    self.selenium.set_page_load_timeout(70)
+    self.selenium.get(login_url)
+
+    # 3️⃣ Fill login credentials
+    self.selenium.find_element(By.NAME, "login").send_keys("bugbug")
+    self.selenium.find_element(By.NAME, "password").send_keys("secret")
+    self.selenium.find_element(By.NAME, "login_button").click()
+
+    # 4️⃣ Wait until login redirects (so we’re not stuck on verify-email)
```

Ruff (0.14.4)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)